boards/arm/nucleo_wl55jc/nucleo_wl55jc.dts

```diff
@@ -135,6 +135,13 @@
 	status = "okay";
 };
 
+
+&dac1 {
+	pinctrl-0 = <&dac_out1_pa10>;
```

Comment:
Please add chosen pin to docs also for this board.

Reply:
done